static/app/views/replays/detail/timestampButton.tsx

```diff
@@ -22,18 +26,52 @@ export default function TimestampButton({
   startTimestampMs,
   timestampMs,
 }: Props) {
+  const {timestampType} = useReplayContext();
+  const timestampMsAsTime = new Date(timestampMs).getTime();
```

Comment:
This statement should be unneccessary. `getTime()` will convert a `Date` into number of milliseconds since the epoch, but `timestampMs` should already have that same value.